pysb/bng.py

```diff
@@ -116,6 +119,11 @@ def __enter__(self):
     def __exit__(self):
         return
 
+    def _delete_tmpdir(self):
+        for f in os.listdir(self.base_directory):
+            os.unlink(os.path.join(self.base_directory, f))
+        os.rmdir(self.base_directory)
+
```

Comment:
The explicit delete known files approach is appealing, but I tend to agree that deleting a recently created temp directory not accessible by other users should be safe, and is lower maintenance. Will go with `rmtree`.